src/newcharacter.cpp

```diff
@@ -487,7 +488,7 @@ int player::create(character_type type, std::string tempname)
 
     // Likewise, the asthmatic start with their medication.
     if (has_trait("ASTHMA")) {
-        tmp = item("inhaler", 0, false);
+        tmp = item( "inhaler", 0, item::find_type( "inhaler" )->charges_default() );
```

Comment:
> How?

You need to include "itype.h", you need to get the itype, you need to know what `itype` function to call. All this should be part of the `item` and/or the `itype` class, nobody outside of it should know how to get the default charges. Btw. the `itype` is now done twice (here and in the constructor).

In fact, callers should not even know about whether an item has charges or not. One should be able to say `item foo = item::random(id, turn)` or `item bar = item::factory_default(id)`.
Here `factory_default` creates an item that looks like it comes straight from the factory (no damage, no additional properties, default charges). But this is far out of the scope of this PR.